config/jobs/kubernetes-sigs/cluster-api/cluster-api-periodics-main-upgrades.yaml

```
@@ -0,0 +1,154 @@
+      env:
+        - name: KUBERNETES_VERSION_UPGRADE_FROM
+          value: "v1.17.17"
+        - name: KUBERNETES_VERSION_UPGRADE_TO
+          value: "v1.18.15"
+        - name: ETCD_VERSION_UPGRADE_TO
+          value: "3.4.3-0"
+        - name: COREDNS_VERSION_UPGRADE_TO
+          value: "1.6.7"
+        - name: GINKGO_FOCUS
+          value: "\\[K8s-Upgrade\\]"
+      # we need privileged mode in order to do docker in docker
+      securityContext:
+        privileged: true
+      resources:
+        requests:
+          cpu: 7300m
+  annotations:
+    testgrid-dashboards: sig-cluster-lifecycle-cluster-api
+    testgrid-tab-name: capi-workload-upgrade-1-17-1-18
+
+- name: periodic-cluster-api-e2e-workload-upgrade-1-18-1-19-main
+  interval: 24h
+  decorate: true
+  labels:
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+  extra_refs:
+  - org: kubernetes-sigs
+    repo: cluster-api
+    base_ref: master
+    path_alias: sigs.k8s.io/cluster-api
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20210201-cc9840e-1.19
+      args:
+        - runner.sh
+        - "./scripts/ci-e2e.sh"
+      env:
+        - name: KUBERNETES_VERSION_UPGRADE_FROM
+          value: "v1.18.15"
+        - name: KUBERNETES_VERSION_UPGRADE_TO
+          value: "v1.19.7"
+        - name: ETCD_VERSION_UPGRADE_TO
+          value: "3.4.13-0"
+        - name: COREDNS_VERSION_UPGRADE_TO
+          value: "1.7.0"
+        - name: GINKGO_FOCUS
+          value: "\\[K8s-Upgrade\\]"
+      # we need privileged mode in order to do docker in docker
+      securityContext:
+        privileged: true
+      resources:
+        requests:
+          cpu: 7300m
+  annotations:
+    testgrid-dashboards: sig-cluster-lifecycle-cluster-api
+    testgrid-tab-name: capi-workload-upgrade-1-18-1-19
+
+- name: periodic-cluster-api-e2e-workload-upgrade-1-19-1-20-main
+  interval: 24h
+  decorate: true
+  labels:
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+  extra_refs:
+  - org: kubernetes-sigs
+    repo: cluster-api
+    base_ref: master
+    path_alias: sigs.k8s.io/cluster-api
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20210201-cc9840e-1.19
+      args:
+        - runner.sh
+        - "./scripts/ci-e2e.sh"
+      env:
+        - name: KUBERNETES_VERSION_UPGRADE_FROM
+          value: "v1.19.7"
+        - name: KUBERNETES_VERSION_UPGRADE_TO
+          value: "v1.20.2"
+        - name: ETCD_VERSION_UPGRADE_TO
+          value: "3.4.13-0"
+        - name: COREDNS_VERSION_UPGRADE_TO
+          value: "1.7.0"
+        - name: GINKGO_FOCUS
+          value: "\\[K8s-Upgrade\\]"
+      # we need privileged mode in order to do docker in docker
+      securityContext:
+        privileged: true
+      resources:
+        requests:
+          cpu: 7300m
+  annotations:
+    testgrid-dashboards: sig-cluster-lifecycle-cluster-api
+    testgrid-tab-name: capi-workload-upgrade-1-19-1-20
```

in order to be consistent with the current naming convention in https://testgrid.k8s.io/sig-cluster-lifecycle-cluster-api
```suggestion
    testgrid-tab-name: capi-e2e-main-1-19-1-20
```
@vincepri opinions?